go-controller/pkg/ovn/controller/admin_network_policy/types.go

```diff
@@ -315,7 +325,13 @@ func newBaselineAdminNetworkPolicyState(raw *anpapi.BaselineAdminNetworkPolicy)
 		}
 		banp.egressRules = append(banp.egressRules, banpRule)
 	}
-
+	banp.aclLoggingParams, err = getACLLoggingLevelsForANP(raw.Annotations)
+	if err != nil {
+		addErrors = errors.Wrapf(addErrors, "error: cannot parse anp ACL logging annotation, disabling it for BANP %v - %v",
```

Comment:
same thing here ANP, but shoudl this one be BANP?

Reply:
yeah good catch should be BANP..

Reply:
done.